### PR TITLE
Fix #101: Awkwards gaps

### LINKED ISSUE #101: Fix Awkward Gaps in phone screen
image
image
image
image
Etc... go through each page

### PR DESCRIPTION
**Pull request by jimmyygit**
Fixes a bunch of little things that affect gaps on mobile screens. 
Also addresses ultra-wide viewports for TextAndPic
View commits for more details

image
image

### CODE REVIEW

**Comment by functionstackx** [comment]
@jimmyygit can you paste some screenshots on yr PR description plz

**Comment by functionstackx** [comment]
this looks super werid. the large block of text:

### Suggestions
- make blocks of text less

## 16x9
image

## ultrawide
image

## mobile (iPhone X)
image

File: components/ResilienceResource/MindfulnessExercises.js [hunk 1]
```diff
@@ -5,8 +5,12 @@ import ColourBoxContainer from './ColourBoxContainer'
 import styles from './MindfulnessExercises.module.css'
 
 function MindfulnessExercises({ exercises }) {
+  let classStyles = []
+  classStyles.push(styles.MindfulnessExercises)
+  classStyles.push('container')
+  classStyles = classStyles.join(' ')
```
Comment (LemonFace0309) [inline components/ResilienceResource/MindfulnessExercises.js:11]: lol what's the purpose of push class styles when you can just initialize them in the array?

**Review by LemonFace0309 — APPROVED** [review]

File: components/ResilienceResource/MindfulnessExercises.js (hunk 1 above)
Comment (jimmyygit) [reply components/ResilienceResource/MindfulnessExercises.js:11]: yea I find that annoying too. It's cuz they're different types.. you have to combine objects and string literals

**Pull request merged**